### DISCUSSION TITLE
How can I start personalizing the frontend?

### DISCUSSION THREAD

**Discussion by guille-amarilla** [GetDKAN/dkan#4127] [Q&A]
Hello, I want to personalize the frontend page, for example, creating my own Dataset Topics, adding some logos, changing labels, etc. How can I achieve that? If there's a tutorial or something similar it would be so much help. Thanks in advance.

**Comment by janette** [comment]
@guille-amarilla if you set up your site either using the ddev-dkan steps or manually, I assume you have a docroot/frontend directory. All of your custom changes should happen in **docroot/frontend/src/**
See: https://github.com/GetDKAN/data-catalog-app/blob/main/README.md#structure-of-the-app

This frontend code is pulling in components from https://github.com/GetDKAN/data-catalog-components. If you need to make a change to a component in the upstream, copy it to your frontend/src/components directory, make your changes and update the template file to pull in your modified component.
